internal/engine/extractors.go

```diff
@@ -16,3 +19,64 @@ func extractImageAndK8sTargets(specs []model.TargetSpec) (iTargets []model.Image
 	}
 	return iTargets, kTargets
 }
+
+// Check to see if we have k8s targets.
+func extractK8sTargets(specs []model.TargetSpec) []model.K8sTarget {
+	kTargets := make([]model.K8sTarget, 0)
```

Comment:
Why do we bother setting the length to 0 here?

Reply:
it's a required argument? i could understand tweaking it if this was a performance-sensitive codepath, but i don't think it matters much here. what would you do instead?

Reply:
I guess I normally do `ktargets := []model.K8sTarget{}` so that's why it looked weird. This SGMT though!